resources/logging/charts/fluentbit/templates/serviceentry.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.externalServiceEntry }}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: {{ template "fluent-bit.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "fluent-bit.metaLabels" . | trim | nindent 4 }}
+  annotations:
+    {{- if .Values.globalAnnotations }}
+      {{- toYaml .Values.globalAnnotations | trim | nindent 4 }}
+      {{- end }}
+spec:
+  hosts:
+  {{- toYaml .Values.externalServiceEntry.hosts | trim | nindent 4 }}
+  location: MESH_EXTERNAL
+  ports:
+  {{- toYaml .Values.externalServiceEntry.ports | trim | nindent 4 }}
```

Same reason as before.
```suggestion
  {{- tpl (.Values.externalServiceEntry.ports | nindent 4) . }}
```